Corrfunc/tests/test_theory.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python
+
+from os.path import dirname, abspath, join as pjoin
+
+import pytest
+import numpy as np
+
+from Corrfunc.tests.common import gals_Mr19
+from Corrfunc.tests.common import (check_against_reference,
+                                   check_vpf_against_reference)
+from Corrfunc.tests.common import generate_isa_and_nthreads_combos
+
+
+@pytest.mark.parametrize('isa,nthreads', generate_isa_and_nthreads_combos())
+def test_DD(gals_Mr19, isa, nthreads):
+    from Corrfunc.theory import DD
+    
+    boxsize = 420.
+    binfile = pjoin(dirname(abspath(__file__)),
+                     "../../theory/tests/", "bins")
+    autocorr = 1
+    periodic = 1
+    
+    x,y,z,w = gals_Mr19
+    results_DD = DD(autocorr, nthreads, binfile, x, y, z,
+                            weights1=w, weight_type='pair_product',
+                            periodic=periodic, boxsize=boxsize,
+                            output_ravg=True, verbose=True,
+                            isa=isa)
```

Comment:
Whitespace issues here - the second line of parameters should probably align with the opening bracket on the first line (or whatever PEP recommends). Similar alignment are there elsewhere - will you please fix if possible. 

(We really need to get a linter into our CI/pre-commit)

Reply:
Yeah, there's a lot of whitespace issues! I clearly need to upgrade to an editor that will flag PEP8 issues on-the-fly. But I think the codebase is littered with whitespace problems, and it's probably more efficient if we do a big PEP8 linting pass on the entire codebase, rather than playing whack-a-mole each time we open a PR. So I may leave this for now, if that's okay.